polytracker/taint_dag.py

```diff
@@ -120,23 +121,23 @@ def __init__(self, buffer: mmap) -> None:
         self.source_index_bits = 8
         self.source_offset_mask = (1 << 54) - 1
 
-        self.buffer = buffer
+        self.buffer = mmap(file.fileno(), 0, prot=PROT_READ)
```

Comment:
Just out of curiosity, why the move of mmap into the init-function?

Reply:
It's because I had a bunch of places where I've copy-pasted `open()` -> `mmap()` -> `TDFile()` code. So I moved at least the `mmap()` into the `__init__`.